content/en/docs/distributions/minikf/minikf-vagrant.md

```diff
@@ -26,8 +26,7 @@ Join the discussion on the
 [#minikf](https://kubeflow.slack.com/messages/CGRKM3N0G/) Slack channel,
 ask questions, request features, and get support for MiniKF.
 
-To join the Kubeflow Slack workspace, please [request an
-invite](https://kubeflow.slack.com/join/shared_invite/zt-cpr020z4-PfcAue_2nw67~iIDy7maAQ).
+To join the Kubeflow Slack workspace, please see [here](/docs/about/community/#slack).
```

Comment:
NIT: Usually we don't use `this`/`here`/etc. when adding hyperlink. So the sentence here can be:

Refer to [Community Slack Guide](/docs/about/community/#slack) for joining the Kubeflow Slack workspace.

Reply:
Done in https://github.com/kubeflow/website/pull/3007/commits/028e09ad53948f87a1f93675ccc9ae553aba8dc6